modules/configuring-output-format.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * cli/checking-policy-compliance.adoc
+:_content-type: CONCEPT
+[id="configuring-output-format_{context}"]
+= Configuring output format
+
+When you check policy compliance by using the `roxctl deployment check`, `roxctl image check`, or `roxctl image scan` commands, you can specify the output format by using the `-o` option. This option determines how the output of a command is displayed in the terminal.
+
+You can change the output format by adding the `-o` option to the command and specifying the format as `json`, `table`, `csv`, or `junit`.
+
+For example, the following command checks a deployment and then displays the result in `csv` format:
+[source,terminal,subs="+quotes"]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" \
+  deployment check --file =_<yaml_filename>_ \
+  -o csv
+----
+
+[NOTE]
+====
+When you do not specify the `-o` option for the output format, the following default behavior is used:
+
+* The format for the `deployment check` and the `image check` commands is `table`.
+* The default output format for the `image scan` command is `json`. This is the old JSON format output for compatibility with older versions of the CLI. To get the output in the new JSON format, specify the option with format, as `-o json`.
+====
+
+Different options are available to configure the output. The following table lists the options and the format in which they are available.
+
+[%header,cols="1,2,1"]
+|===
+|Option
+|Description
+|Formats
+
+|`--compact-output`
+|Use this option to display the JSON output in a compact format.
+|`json`
+
+|`--headers`
+|Use this option to specify custom headers.
+|`table` and `csv`
+
+|`--no-header`
+|Use this option to omit the header row from the output.
+|`table` and `csv`
+
+|`--row-jsonpath-expressions`
+a|Use this option to specify link:https://github.com/tidwall/gjson[GJSON paths] to select specific items from the output. For example, to get the *Policy name* and *Severity* for a deployment check, use the following command:
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" \
```

Comment:
I'd like to see the `-e` flag dropped in the examples, since we have `ROX_ENDPOINT` variable for that (so either, set the env variable inline or drop it like we do with e.g. the API token).

Reply:
This is not resolved for me. We still use the `-e` flag and not the `ROX_ENDPOINT` variable.

Instead, we use the `ROX_CENTRAL_ADDRESS` variable that has caused confusion in the past. Let's either drop the `-e` flag completely (like we have for all other previous examples or explicitly set `export ROX_ENDPOINT=` beforehand.